src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/breadcrumb.rs

```diff
@@ -0,0 +1,416 @@
+//! This module provides a clickable view for a single breadcrumb.
+
+use crate::prelude::*;
+
+use super::GLYPH_WIDTH;
+use super::HORIZONTAL_MARGIN;
```

Comment:
Its hard to adjust the colors now, as they are using only 2 shades. Lets please create separate color constants for:
    - Selected breadcrumbs
    - Parent breadcrumbs (on the left of the selected one) (this should be also the color of separators on the left)
    - History breadcrumbs (on the right of the selected one) (this should be also the color of separators on the left)
    - Hover of non-selected breadcrumbs color.